vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,98 @@
+
+The root level of the installation directory contains:
+
+* A `vcpkg` directory, the vcpkg database, keeps track of installed packages and files
+* A directory per each [triplet](../users/triplets.md) that has installed packages, contains the
+  packages build output
+
+## `vcpkg` directory
+
+The vcpkg directory contains two subfolders (`info` and `updates`) and a file named `status`:
+
+### `info` directory
+
+Contains the list of files installed by each port, the naming convention for these files is: 
+`<port name>_<version>_<triplet>.list`. The primary purpose of these files is to avoid conflicts
+between ports attempting to install files with the same name. The [`vcpkg
+owns`](../commands/owns.md) can be used to query the information in these files.
+
+### `status` file
+
+The vcpkg database, this file contains an entry for every package installed for every triplet. This
+file is automatically updated by vcpkg and should not be modified manually.
+
+## Triplet directories
+
+The output of each package installation is contained in a triplet-specific directory, for example,
+packages installed for the `x64-windows` triplet are located in `installed/x64-windows` directory.
+
+The layout for the subdirectories inside each triplet directory is the same:
+
+| Subdirectory            | File type                                         | 
+| ----------------------- | ------------------------------------------------- |
+| `bin`                   | Release `.dll` and `.pdb` files                   |
+| `debug/bin`             | Debug `.dll` and `.pdb` files                     |
+| `debug/lib`             | Debug `.lib`, `.so`, and `.a` files               |
+| `debug/lib/manual-link` | Manually-linkable debug `.dll` and `.pdb` files   |
+| `debug/lib/pkgconfig`   | pkgconfig files (`.pc`)                           |
+| `include`               | Header-files (`.h`, `.hpp`, `.hxx`)               |
+| `lib`                   | Release `.lib`, `.so`, and `.a` files             |
+| `lib/manual-link`       | Manually-linkable release `.dll` and `.pdb` files |
+| `lib/pkgconfig`         | Pkgconfig files (`.pc`)                           |
+| `share/<port>`          | Miscellaneous files.                              |
+| `share/pkconfig`        | pkgconfig files (`.pc`)                           |
+| `tools/<port>`          | Executable tools                                  |
+
+
+### `bin` and `debug/bin` directories
+
+On Windows, these directories contain DLL and PDB files for release and debug configuration
+respectively.
+
+### `lib` and `debug/lib` directories
+
+Contain object files, any file with the `.lib`, `.so`, and `.a` extensions is placed in these
+configuration-dependent directories.
+
+### `lib/manual-link` and `debug/lib/manual-link` directories
+
+Contain object files that must be manually linked, these directories are configuration dependent.
+Each port must make sure to place manually-linkable files in these directories explicitly.
```

Comment:
This should expand with a simple example of what qualifies a library as manual link:

```md
For example, if the library is intended to define `main()` for the program, it must be placed in the `lib/manual-link/` directory instead of the `lib/` directory.
```